docs/pages/enroll-resources/application-access/okta/appendix-a-okta-saml-app.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Appendix A. Okta SAML app
+description: A guide for creating and configuring Okta SAML app used as IdP provider for Teleport
+  Auth Connector.
+slug: appendix-a-okta-saml-app
+sidebar_position: 6
+---
+
+This page will guide you how to create a new Okta SAML 2.0 app for *Level 1. SSO* integration. It
```

Comment:
I would find a way to remove the appendices, since they add unnecessary complexity to the docs.

Is there a way to inline this content as a partial within the SSO integration guide, SCIM guide, and user sync guide? Otherwise, it muddies the structure of the Okta Integration with Application Access section, and a user who discovers this guide needs to figure out for themselves how it relates to the other guides mentioned in the introductory paragraph.